lib/gelfd2/chunked_parser.rb

```diff
@@ -40,9 +47,23 @@ def self.parse_chunk(data)
         raise TooManyChunksError, "#{total_number} greater than #{MAX_CHUNKS}" if total_number > MAX_CHUNKS
         @@chunk_map[msg_id][:total_chunks] = total_number.to_i
         @@chunk_map[msg_id][:chunks].merge!({seq_number.to_i => zlib_chunk})
+        @@chunk_map[msg_id][:last_chunk_time] = Time.now.getutc.to_i
         msg_id
       end
     end
 
+    private
```

Comment:
no need for 'private' keyword here, there is already one above previous function